mapflow/functional/layer_utils.py

```diff
@@ -501,40 +501,54 @@ def download_results_callback(self, response: QNetworkReply, processing) -> None
         :param response: The HTTP response.
         :param pid: ID of the inspected processing.
         """
+        self.temp_dir = self.settings.value("tempdir")
```

Comment:
After reloading a pluging everything works without this line.
But at the first start results saving gives an error, initialization temdir differs from app startup temdir, but I can't figure out why?

Reply:
What if "tempdir" is not in settings yet here? There was some function that alerted in case it is not set.

Reply:
It is set. And at the initialization it is the same as self.temp_dir, I checked.
But after app startap "temdir" in settings changes (I haven't been able to find where it might happen).
If plugin was reloaded - tempdirs are the same, but at the first start - no.
Maybe only redefine self.temp_dir under the condition:
`if self.temp_dir != self.settings.value("tempdir")`?